application/cms/views.py

```diff
@@ -107,17 +98,16 @@ def create_measure(topic_slug, subtopic_slug):
     return render_template(
         "cms/edit_measure_version.html",
         form=form,
-        data_source_form=data_source_form,
-        data_source_2_form=data_source_2_form,
         topic=topic,
         subtopic=subtopic,
         measure=Measure(),
         measure_version=MeasureVersion(),
         new=True,
         organisations_by_type=Organisation.select_options_by_type(),
         topics=page_service.get_topics(include_testing_space=True),
-        errors=get_form_errors(forms=(form, data_source_form, data_source_2_form)),
+        errors=get_form_errors(forms=(form,)),
         data_not_uploaded_error=False,
+        data_sources_not_added=False,
```

Comment:
I think negative method/variable names can be confusing. Should we make this `data_sources_added` instead?

Reply:
Just keeping it consistent with what's there already 🤷‍♂  It's weird, sure, but not the end of the world probably. Happy to refactor it and the other variables if you think it's necessary.

Reply:
Happy to leave it for a future refactor.